src/main/resources/assets/computercraft/lua/rom/programs/http/pastebin.lua

```diff
@@ -130,29 +55,19 @@ elseif sCommand == "get" then
         return
     end
 
-    -- GET the contents from pastebin
-    local res = get(sCode)
-    if res then
-        local file = fs.open( sPath, "w" )
-        file.write( res )
-        file.close()
+    local success, msg = pastebin.get(sCode, sPath)
 
+    if success then
         print( "Downloaded as "..sFile )
+    else
+        print(msg)
```

Comment:
Probalby be `printError` or `io.stdderr:write(msg .. "\n")`. Or at least keep the previous "Failed" as an error, and then include the detailed message.

Might be worth making the other messages follow the same format too.